node/Cargo.toml

```diff
@@ -3,113 +3,113 @@ name = "zeitgeist"
-sc-basic-authorship = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-cli = { branch = "polkadot-v0.9.32", features = ["wasmtime"], git = "https://github.com/paritytech/substrate" }
-sc-client-api = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-consensus = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-executor = { branch = "polkadot-v0.9.32", features = ["wasmtime"], git = "https://github.com/paritytech/substrate" }
-sc-keystore = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-rpc = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-rpc-api = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-service = { branch = "polkadot-v0.9.32", features = ["wasmtime"], git = "https://github.com/paritytech/substrate" }
-sc-sysinfo = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-telemetry = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-transaction-pool = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sc-transaction-pool-api = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-api = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-block-builder = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-blockchain = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-consensus = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-core = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-inherents = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-keyring = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-offchain = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-runtime = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-session = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-storage = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-timestamp = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-transaction-pool = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-sp-trie = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
-substrate-frame-rpc-system = { branch = "polkadot-v0.9.32", git = "https://github.com/paritytech/substrate" }
+pallet-asset-tx-payment = { workspace = true, features = ["default"] }
+pallet-transaction-payment = { workspace = true, features = ["default"] }
```

Comment:
Couldn't we just activate `features = ["default"]` on every crate inside this `node/Cargo.toml`? So in that way it would be simpler to grasp to use `default` in all no_std places. But I assume not every crate has the `default` feature available, right?

Reply:
The distinction between client and wasm dependencies was specifically made to avoid having to specify additional features. In most cases this works, however for shared pallets also used in the clients we'll have to specify default either way for any pallet afaik.